library/include/modules/Textures.h

```diff
@@ -46,6 +46,11 @@ DFHACK_EXPORT long getIconsTexposStart();
  */
 DFHACK_EXPORT long getOnOffTexposStart();
 
+/**
+ * Get the first texpos for the pathable 32x32 sprites. It's a 2x1 grid.
+ */
+DFHACK_EXPORT long getPathableTexposStart();
```

Comment:
There's going to be more of these. I think we should name these according to which grid they are designed for to keep their intended usage clear. `getPathableMapTexposStart()`? `getMapPathableTexposStart()`?

Reply:
Good call. 
I can't decide on some things though...

my gut feeling is that this word, be it "Map" or anything else, should be a prefix.
the upside is that all "Map" textures will be listed together in my text editor's autocomplete. 🤔 
the downside is that if your script/plugin has multiple textures, they will not be grouped together by the autocomplete thing. :confused:

in the end i think i'd be more okay with the first case. All "Map" textures should be listed/grouped together, but the more i think about it, the more this seems like a not-so-important problem 🤣 and i've been thinking about it for far too long 😳 

oh yeah-- what about old function names? Should those get a prefix as well?
(also do we call it "Map" or... what else? honestly my alternatives sucked so much i'm not even going to mention them 🤣)

Reply:
Renaming the older ones isn't as important, I feel. the "Interface" layer could be the "default" and "Map" (or whatever) could just follow the new standard.

+1 for making it a prefix, though the verb ("get") should probably still go first. `getMapPathableTexposStart()`? (or if we end up using standard cursors, then this goes away entirely, but it will come up again soon enough since we need tiles for the unsuspend overlay)

Reply:
oh yes yes, the "prefix"+texturename is meant to be sandwiched between `get` and `TextposStart` of course!

i mean, "`get`" is immovable. "`TextposStart`" is very explicative and works well. We can only work on what's between those two.

i'm guessing, like, if a script will need multiple textures, its function-names (or whatever these are called) might look like this:
```
getDesignTextposStart
getDesignButtonsSelectedTextposStart
getDesignButtonsUnselectedTextposStart
getMapDesignTilesTextposStart
```

and... like, in this case the "Map" might look a bit out of place, but in the end i think it's more important to "separate" the map textures from normal textures, than to group *all* of a tool's textures together. 🤔